packages/modelence/src/auth/providers/github.test.ts

```diff
@@ -129,10 +131,11 @@ describe('auth/providers/github', () => {
       } as never);
 
     await handler(
-      { query: { code: 'code' } } as unknown as Request,
+      { query: { code: 'code', state: 's' }, cookies: { oauth_state: 's' } } as unknown as Request,
```

Comment:
Since we already have `authToken` cookie stored in camel case, could you change those cookies to camel case too?

(I know that auth_state is more common)

One more thing, can you split it into authStateGithub and authStateGoogle?